mysticeti-core/src/consensus/universal_committer.rs

```diff
@@ -51,28 +61,34 @@ impl UniversalCommitter {
                 tracing::debug!("Outcome of direct rule: {status}");
 
                 // If we can't directly decide the leader, try to indirectly decide it.
-                if !status.is_decided() {
-                    status = committer.try_indirect_decide(leader, round, leaders.iter());
-                    self.update_metrics(&status, false);
+                if status.is_decided() {
+                    leaders.push_front((status.clone(), true));
```

Comment:
should we have a const DIRECT_DECIDE and INDIRECT_DECIDE instead of true and false? This can help indicate they are only there for metrics reasons rather than protocol logic